src/jquery.mask.js

```diff
@@ -467,6 +472,10 @@
         return this.data('mask').getCleanVal();
     };
 
+    $.fn.maskedVal = function(val) {
```

Comment:
After this... could you help me with a jsfiddle using this feature with angular so I can use it as example?